doc/classes/IP.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="IP" inherits="Object" category="Core" version="3.0-beta">
 	<brief_description>
-		IP Protocol support functions.
+		IP Protocol support functions like DNS resolution.
 	</brief_description>
 	<description>
-		IP contains support functions for the IPv4 protocol. TCP/IP support is in different classes (see [StreamPeerTCP] and [TCP_Server]). IP provides hostname resolution support, both blocking and threaded.
+		IP contains support functions for the IPv4 protocol. TCP/IP support is in different classes (see [StreamPeerTCP] and [TCP_Server]). IP provides DNS hostname resolution support, both blocking and threaded.
```

Comment:
What about TYPE_IPV6?

Reply:
I'll drop the version type, as IPv6 support was added in the meanwhile, and together they are the only relevant versions in >99% of use cases.